Services/UI/OJS.Services.Ui.Business/Validations/Implementations/Contests/ContestValidationService.cs

```diff
@@ -38,17 +57,17 @@ public ValidationResult GetValidationResult((Contest?, int?, UserInfoModel?, boo
             return ValidationResult.Invalid(string.Format(ValidationMessages.Contest.IsExpired, contest.Name));
         }
 
-        if (official &&
+        if (contestActivityEntity.CanBeCompeted &&
             !CanUserCompeteByContestByUserAndIsAdmin(
-                contest,
+                contestActivityEntity,
                 user.IsAdmin,
                 isUserLecturerInContest,
                 allowToAdminAlways: true))
```

Comment:
This business logic seems to be changed. Is there a reason to check first for `CanBeCompeted` instead of the previuos `official`